app/views/volunteers/_volunteer.html.slim

```diff
@@ -6,8 +6,8 @@ tr
   td = volunteer.contact.postal_code
   td = volunteer.contact.primary_email
   td = volunteer.birth_year.year if volunteer.birth_year?
-  td = volunteer.language_skills.first.language_name if volunteer.language_skills.any?
-  td = render 'language_skills/show_inline', language_skills: volunteer.language_skills
+  td = volunteer.native_language.language_name if volunteer.native_language.present?
```

Comment:
there could be no no language_skills records associated at all. They are not a mandatory relation.